app/src/main/java/com/heig/atmanager/Utils.java

```diff
@@ -133,5 +154,14 @@ public static String formatNumber(int number) {
         return (number < 10 ? "0" : "") + number;
     }
 
+    public static String firstLetterCapped(String s){
+        if(s == null)
+            return null;
+        else if(s.length() == 1)
+            return s.toUpperCase();
+        else
+            return s.substring(0,1).toUpperCase() + s.substring(1).toLowerCase();
+    }
+
```

Comment:
I think you could use StringUtils.capitalize no ?
´´´
 StringUtils.capitalize(null)  = null
 StringUtils.capitalize("")    = ""
 StringUtils.capitalize("cat") = "Cat"
 StringUtils.capitalize("cAt") = "CAt"
´´´